src/alarms/notifications/EditNotification.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { withRouter } from "react-router-dom";
+import MDSpinner from "react-md-spinner";
+
+import { NotificationForm } from "./NotificationForm";
+
+class EditNotificationModel extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            currentNotification: null
+        };
+    }
+
+    componentDidMount() {
+        const { match, alarmType } = this.props;
+
+        (async () => {
+            const url = (alarmType === "RASTERS") ? (
+                `/api/v4/rasteralarms/${match.params.id}/`
+            ) : (
+                `api/v4/timeseriesalarms/${match.params.id}/`
+            )
+
+            const currentNotification = await fetch(url, {
+                credentials: "same-origin"
+            }).then(response => response.json());
+
+            this.setState({ currentNotification })
+        })();
+    }
+
+    render() {
+        if (
+            this.state.currentNotification &&
+            this.props.organisations.isFetching === false
+        ) {
+            return <NotificationForm
+                currentNotification={this.state.currentNotification}
+                wizardStyle={false}
+            />;
+        } else {
+            return <div
+                style={{
+                    position: "relative",
+                    top: 50,
+                    height: 300,
+                    bottom: 50,
+                    marginLeft: "50%"
+                }}
+            >
+                <MDSpinner size={24} />
+            </div>
+        }
+
+    }
+}
+
+const mapStateToProps = (state) => {
+    return {
+        organisations: state.organisations,
+        alarmType: state.alarmType
+    };
+};
+
+const App = withRouter(connect(mapStateToProps)(EditNotificationModel));
+
+export { App };
```

Comment:
I think it is confusing that all these screens are called "App" and then need to be renamed when importing them elsewhere, why not just export default so they can be imported as EditNotification?

Reply:
Just to keep it consistent with other screens in the alarm management. I personally don't like to name them this way but rename all of them seems not relevant to the purpose of this ticket.